core/src/test/scala/com/pingcap/tispark/datasource/ErrorSuite.scala

```diff
@@ -0,0 +1,126 @@
+package com.pingcap.tispark.datasource
+
+import com.pingcap.tikv.exception.TiBatchWriteException
+import org.apache.spark.sql.Row
+import org.apache.spark.sql.types.{IntegerType, LongType, StringType, StructField, StructType}
+
+// without TiExtensions
+// will not load tidb_config.properties to SparkConf
+class ErrorSuite extends BaseDataSourceSuite("test_datasource_error") {
```

Comment:
should we rename it as `ExceptionTestSuite`?

Reply:
nice, will rename in next patch